prql-python/Cargo.toml

```diff
@@ -7,6 +7,8 @@ edition = "2021"
 name = "prql_python"
 crate-type = ["cdylib"]
 
```

Comment:
Didnt no that was possible, is this a standard rust / cargo feature ?!

Reply:
Yes! We do it already in `prql-web`!